map/framework.cpp

```diff
@@ -3209,3 +3209,15 @@ void Framework::VizualizeRoadsInRect(m2::RectD const & rect)
     }
   }, kScale);
 }
+
+vector<MwmSet::MwmId> Framework::GetMwmsByRect(m2::RectD const & rect)
+{
+  vector<MwmSet::MwmId> result;
+  if (!m_infoGetter)
+    return result;
+
+  auto countryId = m_infoGetter->GetRegionCountryId(rect.Center());
```

Comment:
This method is quite slow (O(number of countries + size of country boundary * num of candidates).